internal/k8shandler/logstore.go

```diff
@@ -30,7 +32,8 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateLogStore() (err error
 
 		cluster := clusterRequest.Cluster
 
-		if err = clusterRequest.createOrUpdateElasticsearchSecret(); err != nil {
+		if err = clusterRequest.removeElasticsearchIfSecretOwnedByCLO(); err != nil {
```

Comment:
@vparfonov could you please explain upgrade behavior? What happens with already provisioned ES instances for which certs are managed by CLO?  

If am reading this correctly the CLO removes ES if the certs  were managed by CLO. And then a new ES CR is created with cert-management from EO?

Reply:
@pavolloffay I going to check ownership of secret if not ES so i need to remove it and re-create deployments

Reply:
does this imply ~no~ zero time downtime during the upgrade?

Reply:
yep, in this case

Reply:
In this case I am doing something similar for Jaeger https://github.com/jaegertracing/jaeger-operator/pull/1708

Reply:
@vparfonov how does the log-forwarder/cliens (e.g. fluentd?) behave in this case? They will fail to export the data and fail/drop if they cannot export?